WORKSPACE

```diff
@@ -35,6 +35,8 @@ load("@local_config_fuchsia//:workspace.bzl", "fuchsia_base_dependencies")
 fuchsia_base_dependencies(locals = LOCALS)
 load("@local_config_fuchsia//:fuchsia_sdk.bzl", "fuchsia_sdk_dependencies")
 fuchsia_sdk_dependencies(locals = LOCALS)
+load("@fuchsia_clang//:defs.bzl", "register_clang_toolchains")
```

Comment:
load("@local_config_fuchsia//:fuchsia_clang.bzl", "fuchsia_clang_dependencies")
fuchsia_clang_dependencies(locals = LOCALS)

//fuchsia_clang.bzl
load("@gapid//tools/build/rules:repository.bzl", "maybe_repository")
load("@fuchsia_clang//:defs.bzl", "register_clang_toolchains")

def fuchsia_clang_dependencies(locals = {}):
  maybe_repository(
    fuchsia_clang_repository,
    name = "fuchsia_clang",
    locals = locals,
    sdk_root_label = "@fuchsia_sdk",
  )
  register_clang_toolchains()

Reply:
Thanks Chase.  This will be on a different PR.